src/dimension-handler.js

```diff
@@ -14,6 +14,8 @@
 
 (function(scope, testing) {
 
+  var mexp = require('math-expression-evaluator');
```

Comment:
I wasn't sure how you guys would want to handle dependencies, since this would be the first one. Is there a solution that would be preferred over `require` with a module bundler?

One option that comes to mind is something like `typeof require !== 'undefined' ? require ('math-expression-evaluator').eval : eval`, which would accommodate both users like me who can't use `eval` for CSP compliance and users who want the size advantage you mentioned of not including another dependency (either by loading web-animations-js with a script tag or by configuring math-expression-evaluator as an undefined external in their module bundler).